guides/common/modules/proc_adding-azure-details-to-a-compute-profile.adoc

```diff
@@ -26,8 +25,20 @@ Note that the username that you enter for compute profile must be the same that
 . Optional: In the *Custom Script Command* field, enter commands to perform on the virtual machine when the virtual machine is provisioned.
 . Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use.
 The scripts must contain `sudo` at the beginning because {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host and scripts require sudo privileges to be executed.
+. If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. Please refer to the Microsoft documentation links below for more details.
+  - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux[NVIDIA GPU Driver Extension for Linux]
+  - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-windows[NVIDIA GPU Driver Extension for Windows]
 . Optional: If you want to create an additional volume on the VM, click the *Add Volume* button, enter the *Size* in GB and select the *Data Disk Caching* method.
+  - *IMPORTANT*: Note that the maximum number of these disks depends on the VM Size selected. Please refer to the Microsoft documentation links below for more details.
+    - https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-general[General Purpose]
```

Comment:
Perhaps just provide one overall VM link, rather than all these. 
These links might break in time. I'd literally just include this one https://docs.microsoft.com/en-us/azure/virtual-machines and tell users to go look there.